schema.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "event",
+    "type": "object",
+    "properties": {
+ 	 "organizer": {
+        "name": "name",
+        "email": "email"
+  	 },
+  	"id": "number",
+    "name": "event_name",
+    "location": {
+        "description": "zip code of event",
+        "type": "number",
+        "minimum": 0,
+        "maximum": 99999
+    },
```

Comment:
Just to temporarily simplify things I think it'd be easier if we did "location" as just a String instead of another object.

Reply:
Can add back in once our MVP is done

Reply:
sounds like a plan!